docs/plugins.md

```diff
@@ -0,0 +1,33 @@
+# Plugins
+
+Since 0.4.11 go-ipfs has a plugin system for various added functionality.
+
+When IPFS node is created, it will load plugins from `.ipfs/plugins` directory.
+
+### Plugin types
+
+#### IPLD
+IPLD plugins add support for additional LD formats to `ipfs dag` and other IPLD
```

Comment:
`s/additional LD/additional IPLD/`. We don't support other LD formats (JSON-LD etc...).